pca.go

```diff
@@ -0,0 +1,79 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+)
+
+// PrincipalComponents returns the principal component direction vectors of
```

Comment:
The documentation is in general good. Two comments
1) PCA freqently says that the columns need to be zero mean and variance of 1. This variance is not rescaled. This is the correct approach, but I think it should be noted.
2) I don't see anything about the singular values changing from v --> f * v^2. I'm sure this is a standard thing, but I don't see any mention of it in the wikipedia article or in the Stanford machine learning notes. Could you provide a link that explains "variance values" so that users can read what it means?